haskell-overlays/untriaged.nix

```diff
@@ -83,7 +83,8 @@ in self: super: {
   # Packages not yet in 19.03
   semialign = doJailbreak (self.callHackage "semialign" "1" {});
   these = self.callHackage "these" "1" {};
-  these-lens = self.callHackage "these-lens" "1" {};
+  these-lens = doJailbreak (self.callHackage "these-lens" "1" {});
+  # these-lens wants an old lens version for no particular reason
```

Comment:
Can you open a PR to `these-lens` loosening bounds sufficiently? That will help this jailbreak go away eventually...

Reply:
i have been informed this is indeed "bad vibes" and done this. once this lands i think the jb can go away https://github.com/isomorphism/these/pull/134

Reply:
yep, https://github.com/isomorphism/these/pull/134